# Example camera parameters (for Jack O'Quin's Unibrain Fire-i BCL 1.2)

#guid: 08144361026320a0 (defaults to first camera found)
camera_info_url: file:///home/joq/svn/camera_drivers/camera1394/launch/unibrain_calibration.yaml
video_mode: 640x480_rgb
fps: 15

# these all default to automatic settings:
#brightness: 511
#exposure: 256
#gain: 150
#shutter: 12
#whitebalance: "2000 2000"
